include/sys/spa.h

```diff
@@ -555,13 +675,14 @@ _NOTE(CONSTCOND) } while (0)
 		    DVA_GET_ASIZE(&bp->blk_dva[1]) / 2)			\
 			copies--;					\
 		len += func(buf + len, size - len,			\
-		    "[L%llu %s] %s %s %s %s %s %s%c"			\
+		    "[L%llu %s] %s %s %s %s %s %s %s%c"			\
```

Comment:
i printed some random bps that were coming through the zio layer and they use about 250 bytes so ill add another few bytes in for safety.